deploy/ray-operator/config/crd/bases/ray.io_rayclusters.yaml

```diff
@@ -791,6 +794,12 @@ spec:
                     description: 'Head service suffix. So head can be accessed by
                       domain name: {namespace}.svc , follows Kubernetes standard.'
                     type: string
+                  idList:
+                    description: create operation should have no idList (generate
```

Comment:
Explain more about what idList is designed for.

Reply:
Added comments in the latest commit.